File: 50 Coding Challenge Part I/C/challenge7.c
// Coding challenge 7: Calculate the sum of odd numbers greater than
// 10 and less than or equal to 30

#include <stdio.h>

int main(int argc, char *argv[]) {
	int sum = 0;

	for(int i = 11; i <= 30; i += 2) {
		sum += i;
	}

	printf("Sum = %d\n", sum);

	return 0;
}

File: 50 Coding Challenge Part I/dart/lib/challenge7.dart
// Coding Challenge 7: Calculate the sum of odd numbers greater than 10
// and less or equal than 30
// @author: mofirojean

int calculateSumWithCondition() {
  int sum = 0;
  for (var i = 10; i <= 30; i++) {
    if (i % 2 != 0) {
      sum += i;
    }
  }
  return sum;
}
